developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -177,19 +198,18 @@ import ManageDataCodeV3 from '!!raw-loader!/_includes/code/howto/manage-data.rea
   </TabItem>
 </Tabs>
 
-### Collections, classes and schemas
+### Nomenclature changes
 
-The focus on [collections](/developers/weaviate/config-refs/schema) in the v4 client is related to some corresponding changes in Weaviate Core:
+Some of the nomenclature within the Weaviate ecosystem is changing, and the client nomenclature has changed accordingly:
 
-- Old-style "class" objects are "collections" now.
-- An old "schema" is a "collection configuration" now.
-- A "property schema" is a sub-set of the collection configuration.
+- A "class", in the sense of a set of objects, is a "collection".
+- The Weaviate "schema" can be accessed through "collection configuration".
```

Comment:
This doesn't distinguish schema as used with properties v. the old meaning which includes all of the collection configuration. 

(See Sebastian's notes from last week)

Reply:
Actually, neither approach explains the topic correctly.

how about something like this:
> - A "Class" is now called a "Collection". A collection stores data objects together with their vector embeddings.
> - A "Schema" is now called a "Collection Configuration", a set of settings that define collection name, used vectorizers, index configuration, property defintions, etc.
> - We introduced "Property Schema", which refers to the definition of property names, their datatypes and other property configurations.

Reply:
Fair. I consider this to be something that deserves a site-wide review, with a coordinated change and some dev education. Perhaps worth tackling in a separate PR.

Reply:
I am going to leave out "property schema" as that bigram isn't used anywhere else.